tests/e2e/scenarios/test_journey_coverage.py

```
@@ -315,6 +325,173 @@ def _assert_rust_evidence(case: JourneyCase, evidence: CargoEvidence) -> None:
+        f"{case.case_id}: expected_count does not gate the provider mutation count"
+    )
+
+    test_body = _rust_function_body(source, case.evidence.test)
+    reachable_bodies = [test_body]
+    for delegate in set(re.findall(r"\b([a-z][A-Za-z0-9_]*_impl)\s*\(", test_body)):
+        if delegate == address.assertion:
+            continue
+        try:
+            reachable_bodies.append(_rust_function_body(source, delegate))
+        except AssertionError:
+            pass
```

Ruff (0.16.0)

[warning] Use `contextlib.suppress(AssertionError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(AssertionError): ...`

(SIM105)